kubemarine/patches/reinstall_etcdctl_thirdparty.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021-2023 NetCracker Technology Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from textwrap import dedent
+
+from kubemarine.core.action import Action
+from kubemarine.core.patch import RegularPatch
+from kubemarine.core.resources import DynamicResources
+from kubemarine.thirdparties import install_thirdparty
+
+
+class TheAction(Action):
+    def __init__(self) -> None:
+        super().__init__("Reinstall etcdctl thirdparty")
+
+    def run(self, res: DynamicResources) -> None:
+        logger = res.logger()
+        cluster = res.cluster()
+        if cluster.inventory['services']['cri']['containerRuntime'] == 'docker':
+            logger.info("Docker cri is used, updating etcdctl thirdparty is not needed")
+            return
```

Comment:
Why not needed for docker? I think it may be relevant for both CRI

Reply:
Fixed in https://github.com/Netcracker/KubeMarine/pull/651/commits/38b8bc048059c7fefb6cae80fc0bc3f949a97742